### PR TITLE
found acnh items and organized into files

### LINKED ISSUE #6: locate or create database/dictionary of animal crossing items with prices

### PR DESCRIPTION
**Pull request by jeremyarde**
- found some items and filtered into new data files
- added some requirements for python modules required
- added test photos (cropped and uncropped, three image of same scene)
- code to test out ocr on photos in test_photos folder

### CODE REVIEW

File: backend/README.md [hunk 1]
```diff
@@ -13,11 +13,47 @@ If the version of python is 3.6+, then great! Otherwise download python 3.6+.
...
+
+## OCR Setup
+
+We are using the `pytesseract` python module to perform OCR.
+
+In order to run OCR you need to download and install the OCR binary for your machine from the following location:
+
+> https://github.com/UB-Mannheim/tesseract/wiki
+
+In order for the tesseract exe to be found in your python files, we need to either add the exe location to your environment variables, or change the following line of code to point to where the exe is located:
```
Comment (jadasc) [inline backend/README.md]:
``` suggestion

In order for the Tesseract exe to be found in your python files, we need to either add the exe location to your environment variables, or change the following line of code in `\backend\text_extractor.py` to point to where the exe is located:
```

File: backend/text_extractor.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import glob
+
+import pytesseract
+from PIL import Image
+# class TextExtractor:
+#     def extract_text(img):
+pytesseract.pytesseract.tesseract_cmd = r"C:\Users\Jeremy\AppData\Local\Tesseract-OCR\tesseract.exe"
```
Comment (jadasc) [inline backend/text_extractor.py:7]: Is it possible to get this value from a different file or source that isn't in source control?

Comment (jeremyarde) [reply backend/text_extractor.py:7]: Yes, we can get this from a command line argument, a config file, etc.

**Review by jeremyarde — COMMENTED** [review]
approved!

**Review by jadasc — APPROVED** [review]

**Pull request merged**